docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -233,13 +235,16 @@ Alternatively, you can configure the tenant `tenant-a` directly in `application.
 [source,properties]
 ----
 # Default tenant configuration
-%prod.quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus
+%prod.keycloak.url=http://localhost:8180
```

Comment:
@jedla97 Please keep a single property here too

Reply:
@jedla97 In general, we'd like to avoid highlighting Keycloak may be involved as there is still some perception that quarkus-oidc is for Keycloak users only

Reply:
Done

> @jedla97 In general, we'd like to avoid highlighting Keycloak may be involved as there is still some perception that quarkus-oidc is for Keycloak users only

Thanks I didn't know that